ionic-angular-template/src/auth/sign-in/sign-in.component.scss

```diff
@@ -0,0 +1,7 @@
+form {
+    padding: 5%;
```

Comment:
What happened with the classes from the main stylesheet file, where you have styleForm and etc?